.github/CODEOWNERS

```
@@ -1,35 +1,38 @@
-/packages/@unimodules                           @sjchmiela
+/packages/@unimodules/core                      @sjchmiela @tsapeta
+/packages/@unimodules/react-native-adapter      @sjchmiela @tsapeta
```

Wouldn't this work too?
```suggestion
/packages/@unimodules                           @sjchmiela @tsapeta
```

---

But why? :P I'd prefer having each package listed here separately

---

If we ever add a new package to `@unimodules` we wouldn't have to customize this file. I'm also ok with leaving this as it is.